cosmos/seda.json

```diff
@@ -0,0 +1,53 @@
+{
+  "chainId": "seda-1",
+  "chainName": "SEDA",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/seda/chain.png",
+  "rpc": "https://rpc.seda.xyz",
+  "rest": "https://lcd.seda.xyz",
+  "nodeProvider": {
+    "name": "SEDA",
+    "email": "info@seda.xyz",
+    "website": "https://www.seda.xyz/"
+  },
```

Comment:
Could you remove nodeProvider object? It's not required to a natively supported chain.
```suggestion
```